src/main/java/com/um/inventoryservice/BusinessLayer/OrderServiceImpl.java

```diff
@@ -29,6 +29,7 @@ public Flux<OrderDTO> getOrdersByStockItemId(String stockItemId){
     public Mono<OrderDTO> insertOrder(Mono<OrderDTO> orderDTOMono) {
         return orderDTOMono
                 .map(EntityDTOUtil::toEntity)
+                .doOnNext(o -> o.setOrderId(EntityDTOUtil.generateOrderString()))
```

Comment:
Is this a functionality we eventually want to modify so the data can be entered manually and not randomly?

Reply:
I think it would stay random because they don't want to have to enter an id by themselves manually plus it would mean they could possibly have duplicates one day.